### PR TITLE
replaces dependencies on tendermint and cosmos-sdk to our forks

### LINKED ISSUE #1: Replace dependency on official tendermint/cosmos libraries with our forks
## Feature description 
To ensure compatibility with our Celestia chain, we will need to depend on our forks of [tendermint](https://github.com/celestiaorg/celestia-core) and [cosmos-sdk](https://github.com/celestiaorg/cosmos-sdk), so that all transactions are handled correctly.

## Implementation proposal
Instead of making change on [juno](https://forbole/juno) and replace all imports with ours, and related work. For now, let's replace the dependencies directly in the current repository with ours. It's easier and more maintainable.

### PR DESCRIPTION
**Pull request by rach-id**
## Description

Closes: https://github.com/celestiaorg/bdjuno/issues/1

replaces dependencies on tendermint and cosmos-sdk to our forks.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,4 +24,6 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 
 replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
-replace github.com/tendermint/tendermint => github.com/forbole/tendermint v0.34.13-0.20210820072129-a2a4af55563d
+replace github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v0.34.14-celestia.0.20211121093520-199a1e4232ac
```
Comment (evan-forbes) [inline go.mod]: while we're at it, we should use the commit after the one listed here, as it has event tracking for malleated txs
266919cced265eb629fd51fdc7ba264f4c3a499a

File: go.mod [hunk 2]
```diff
@@ -24,4 +24,6 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
...
+
+replace github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v0.44.1-celestia
```
Comment (evan-forbes) [inline go.mod:29]: might also want to wait to merge until we update our fork of the sdk

**Review by evan-forbes — COMMENTED** [review]
:+1:

File: go.mod (hunk 1 above)
Comment (rach-id) [reply go.mod]: Changed it to: `github.com/celestiaorg/celestia-core v0.34.14-celestia` which should reflect the latest. what do you think ?

File: go.mod (hunk 2 above)
Comment (rach-id) [reply go.mod:29]: sure

File: go.mod (hunk 1 above)
Comment (evan-forbes) [reply go.mod]: I don't think that tag includes that change, we could however create a new tag that does. We might also be able to use the branch `v0.34.x-celestia latest`

https://github.com/celestiaorg/celestia-core/commits/v0.34.14-celestia

we'll be changing this soon, so it doesn't matter that much.

Comment (rach-id) [reply go.mod]: cool then, let's wait until we have all the changes

**Comment by rach-id** [comment]
Will close this one as these changes are obsolete for this new PR: https://github.com/celestiaorg/bdjuno/pull/9

**Pull request closed without merging**